### ISSUE TITLE
Failed test's error points to Chai

### ISSUE THREAD

**Issue by Reinmar** [litixsoft/karma-mocha-reporter#81]
See e.g. https://travis-ci.org/ckeditor/ckeditor5-engine/builds/173847164#L7972:

```
Error: Uncaught AssertionError: expected 1 to equal 2 (/home/travis/build/ckeditor/ckeditor5-engine/node_modules/chai/chai.js:206)
```

This is pretty useless if a test have more assertions. In the [screenshot](https://github.com/litixsoft/karma-mocha-reporter/raw/master/demo/screen.png) I can see that with some other assertion lib it works fine, so perhaps it's a problem with Chai. But what could help (and in fact, be pretty useful in general) was if a longer stack trace was logged in case of a failed assert.

**Comment by 4kochi** [comment]
There is no limitation in the length of the stack trace. The reporter just displays the value coming from karma. So I guess this must be a problem specific to chai. I have tested it locally with the test included in this repository and they work fine.

<img width="752" alt="screen" src="https://cloud.githubusercontent.com/assets/1742426/20056291/aa19769e-a4e5-11e6-8453-848c048026c9.png">

**Comment by Reinmar** [comment]
Hmm... We'll have to dig into this. I've got some theories and I'm curious now which one of them it is – Webpack, source maps, different assertion function used. I'll let you know if we find something :).

Thanks! :)

**Comment by rosston** [comment]
I've had similar frustrations with stack traces using the same tech stack (except browserify instead of webpack). I don't remember the particulars of it now, but I traced it back to karma (or maybe karma-mocha?) itself messing with the stack traces.

**Comment by Reinmar** [comment]
I debugged this. It turns out that we had a broken test which caused an async error to be thrown so it was caught by mocha's native `onerror` listener. So it has the stack of length 2 and this is quite understandable now (or at least more acceptable, taken the sourcemaps) that we got this a bit useless error. I'm only unsure why it's 1!=2 assertion, but I won't have time to debug it further now, so I must live with this :D.

Thanks for the help! :)
